### ISSUE TITLE
Argument is deprecated

### ISSUE THREAD

**Issue by IanMoroney** [GoogleCloudPlatform/cloud-foundation-fabric#2506]
I understand this is fixed now according to [this pr](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/2502), just tracking here for completion.
If you could close this in the next release that would be great 👍 
```
│ Warning: Argument is deprecated
│ 
│   with module.domain-projects["myproject"].google_project.project[0],
│   on .terraform/modules/domain-projects/cloud-foundation-fabric-32.0.0/modules/project/main.tf line 53, in resource "google_project" "project":
│   53:   skip_delete         = var.skip_delete
│ 
│ skip_delete is deprecated and will be removed in 6.0.0. Please use deletion_policy instead. A skip_delete value of false can be changed to a deletion_policy value
│ of DELETE and a skip_delete value of true to a deletion_policy value of ABANDON for equivalent behavior.
```

**Comment by juliocc** [comment]
This is already fixed. What is the ask here?

**Comment by IanMoroney** [comment]
it's not released yet, right? it wasn't included in v33.0.0

**Comment by ludoo** [comment]
> it's not released yet, right? it wasn't included in v33.0.0

We don't keep issues open for features already implemented, which will go in the next release. :) It's not needed (they will go in the release regardless), and it would make issues unusable to track actual bugs/FRs.
